text_collector_examples/smartmon.sh

```diff
@@ -141,5 +161,9 @@ for device in ${device_list}; do
   # Get the SMART information and health
   /usr/sbin/smartctl -i -H -d "${type}" "${disk}" | parse_smartctl_info "${disk}" "${type}"
   # Get the SMART attributes
-  /usr/sbin/smartctl -A -d "${type}" "${disk}" | parse_smartctl_attributes "${disk}" "${type}"
+  if [[ "${type}" == 'sat' ]] ; then
```

Comment:
I might suggest using a case statement here, to avoid trying to process unknown types as scsi.

Reply:
you are right. I will change and add to this PR. thanks